### ISSUE TITLE
Build breaking on Github actions after docs missing in v5.4.0

### ISSUE THREAD

**Issue by radmanz** [iamstevendao/vue-tel-input#265]
# Issue Description
Github actions have stoped passing due to the following error

```sh
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vue-tel-input@5.4.0 postinstall: `cd docs && npm ci --prefer-offline`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vue-tel-input@5.4.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-06-20T23_36_37_223Z-debug.log
Error: Process completed with exit code 2.
```

# Current Setup
My package.json consisted of the following line, where v5.2.0 and above are suitable for my application.
```sh
"vue-tel-input": "^5.2.0",
```

# Temporary Fix
I have removed the ^ char from the package.json and github actions can now build my application.

Would be great to continue using versions greater than v5.2.0 for my application.

**Comment by tafel** [comment]
Same problem here. The temporary fix is to ignore scripts (`npm install vue-tel-input --ignore-scripts`). But it's annoying because I need to add this all along the deployment process.

    $ npm remove vue-tel-input
    $ npm install vue-tel-input

    > vue-tel-input@5.4.0 postinstall /app/packages/clients-front/node_modules/vue-tel-input
    > cd docs && npm ci --prefer-offline

    sh: 1: cd: can't cd to docs

    npm ERR! code ELIFECYCLE
    npm ERR! errno 2
    npm ERR! vue-tel-input@5.4.0 postinstall: `cd docs && npm ci --prefer-offline`
    npm ERR! Exit status 2
    
    $ npm install vue-tel-input --ignore-scripts
    + vue-tel-input@5.4.0
    added 2 packages from 7 contributors, removed 7 packages and audited 1698 packages in 11.817s

**Comment by irvv17** [comment]
I have the same Issue, works with.

`"vue-tel-input": "^5.2.0",`

**Comment by iamstevendao** [comment]
Sorry guys, it should be fixed in [v5.5.0](https://github.com/iamstevendao/vue-tel-input/releases/tag/v5.5.0)
